airflow/www/views.py

```diff
@@ -3242,6 +3258,10 @@ def gantt(self, dag_id, session=None):
         wwwutils.check_import_errors(dag.fileloc, session)
 
         dt_nr_dr_data = get_date_time_num_runs_dag_runs_form_data(request, session, dag)
+
+        if "error" in dt_nr_dr_data:
+            return Response(dt_nr_dr_data["error"], mimetype="text/plain", status=400)
```

Comment:
This can be dropped I think?

Reply:
Sorry about that, I removed one instance and forgot this one. Should be fixed now.